airflow/models/baseoperator.py

```diff
@@ -566,7 +566,20 @@ def priority_weight_total(self):
     @cached_property
     def operator_extra_link_dict(self):
         """Returns dictionary of all extra links for the operator"""
-        return {link.name: link for link in self.operator_extra_links}
+        from airflow.plugins_manager import operator_extra_links
+
+        op_extra_links_from_plugin = {}
+        for ope in operator_extra_links:
+            if ope.operator_name == self.__class__.__name__:
```

Comment:
Shall we also include the module name here? It is possible that the user create an operator with the same name but registered in a different package through plugin?

Reply:
Updated the PR